Dockerfile

```diff
@@ -1,33 +1,106 @@
-FROM node:20.2.0-slim
+FROM node:20.2.0-slim as compiler
+
+#RUN apt-get update && apt-get -y install git wget openssl
+
 WORKDIR /app
 
-# Define node.js environment variables
+#RUN git clone https://github.com/ajnart/homarr.git .
+COPY . .
+
+RUN yarn install
+COPY .env.example .env
+RUN yarn build
+
+
+FROM node:20.2.0-alpine3.18
```

Comment:
You build on a glibc host, then copy over to a musl one?

This can introduce problems if not careful if you copy over content that was tied to a specific environment like this.